### PR TITLE
Create member

### PR DESCRIPTION
**Pull request by Tuuleh**

### CODE REVIEW

File: app/workers/queue_listener.rb [hunk 1]
```diff
@@ -104,8 +109,14 @@ def create_payment(params)
     client.create_recurring_payment(params[:params])
   end
 
-  def create_payment(params)
-    client.create_recurring_payment(params[:params])
+  def subscribe_member(params)
+    page_name = ENV['AK_SUBSCRIPTION_PAGE_NAME']
+    byebug
```
Comment (Tuuleh) [inline app/workers/queue_listener.rb]: @osahyoun could you help me out figure out this one. I have an issue where `ENV['AK_SUBSCRIPTION_PAGE_NAME']` is always empty string. I have set it under test in `config/env.yml` as `'registration'`, and even if I explicitly set it in the spec, it's still empty string. You can run `rspec spec/requests/subscriptions_spec.rb:19` and you'll hit the `byebug` in `subscribe_member`, where if you `pp ENV` you might run into the same issue. Tell me if you do.

File: config/env.yml.template [hunk 2]
```diff
@@ -3,3 +3,4 @@ AK_PASSWORD: 'password'
 AK_USERNAME: 'username'
 AK_HOST: 'https://act.sumofus.org/'
 HPC_RULE_ID: '22'
+AK_SUBSCRIPTION_PAGE_NAME: 'registration'
```
Comment (osahyoun) [inline config/env.yml.template]: 👍 courteous.

File: spec/requests/actions_spec.rb [hunk 3]
```diff
@@ -121,7 +121,7 @@
         before do
           data[:donationpage][:payment_account] = 'PayPal EUR'
           VCR.use_cassette('donation_push_paypal') do
-            puts post '/message', params
```
Comment (osahyoun) [inline spec/requests/actions_spec.rb]: 😆

File: circle/test.env [hunk 4]
```diff
@@ -4,3 +4,4 @@ AK_HOST=https://act.sumofus.org/rest/v1
 AK_USERNAME=ak_username
 AK_PASSWORD=ak_password
 HPC_RULE_ID=22
+AK_SUBSCRIPTION_PAGE_NAME='registration'
```
Comment (osahyoun) [inline circle/test.env]: 🌷

**Pull request merged**